### PR TITLE
encounter custom attribute patch add operation

### PR DESCRIPTION
**Pull request by Anurag-Balannagari**
Description
----
encounter custom attribute patch add operation and chargeitem operation definition 

Screen Shot 2022-06-03 at 9 53 22 AM
Screen Shot 2022-05-31 at 7 55 57 AM

PR Checklist
----
- [x] Screenshot(s) of changes attached before changes merged.
- [ ] Screenshot(s) of changes attached after changes merged and published.

### CODE REVIEW

**Review by ahamdany — APPROVED** [review]

File: lib/resources/r4/encounter_patch.yaml [hunk 1]
```diff
@@ -111,3 +111,36 @@ operations:
         <li> <a href="https://fhir-ehr.cerner.com/r4/StructureDefinition/payment-collection-status?_format=json">Payment Collection Status</a></li>
         <li> <a href="https://fhir-ehr.cerner.com/r4/StructureDefinition/estimated-financial-responsibility-amount?_format=json">Estimated Financial Responsibility Amount</a></li>
       </ul>
+
+  - name: add-extension
+    path: /extension/-
+    operation: add
+    type: List of Extension
```
Comment (exopsRamba) [inline lib/resources/r4/encounter_patch.yaml:118]: This shouldn't be a list, add only supports a single extension.

File: lib/resources/r4/encounter_patch.yaml [hunk 2]
```diff
@@ -111,3 +111,36 @@ operations:
...
+              }
+            ],
+            "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/custom-attribute"
+          }
+        }
+      ]
+    note: >
+      <ul>
+        <li> The custom-attribute id field cannot be set.</li>
+        <li> Add operation is currently supported for <a href="https://fhir-ehr.cerner.com/r4/StructureDefinition/custom-attribute?_format=json">Custom Attribute Extension</a></li>
```
Comment (exopsRamba) [inline lib/resources/r4/encounter_patch.yaml:145]: nit: let's put 'currently **only** supported for...'

File: lib/resources/r4/encounter_patch.yaml (hunk 1 above)
Comment (Anurag-Balannagari) [reply lib/resources/r4/encounter_patch.yaml]: updated here https://github.com/cerner/fhir.cerner.com/pull/849/commits/82c12687b37d80a0d2e24607c7931ee43adddcbb#diff-a0c53684e260623feb3826256c549a1eeb448b54febcc11c63fa0c15d4e86b20R118

File: lib/resources/r4/encounter_patch.yaml (hunk 2 above)
Comment (Anurag-Balannagari) [reply lib/resources/r4/encounter_patch.yaml]: updated here https://github.com/cerner/fhir.cerner.com/pull/849/commits/82c12687b37d80a0d2e24607c7931ee43adddcbb#diff-a0c53684e260623feb3826256c549a1eeb448b54febcc11c63fa0c15d4e86b20R145

**Review by exopsRamba — APPROVED** [review]

**Review by gklsan — APPROVED** [review]

**Pull request merged**